Sources/Puree/Logger.swift

```diff
@@ -19,43 +19,42 @@ public final class Logger {
     }
 
     private let configuration: Configuration
+    private let dispatchQueue = DispatchQueue(label: "com.cookpad.Puree.Logger.\(arc4random())", qos: .background)
```

Comment:
Why should this be random?

Reply:
I misunderstood.
I added this for every loggers works in their own thread but it was not needed.